guides/deployments.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Deployments"
+description: "Troubleshoot your deployments"
+icon: "boxes"
+---
+
+Your documentation site automatically deploys when you push changes to your connected repository. If your latest changes are not appearing on your live site, follow these troubleshooting steps.
```

Comment:
Lets add more context here! If automatic updates arent working they probably dont have the github app installed. maybe walk them through that and if that is properly connected and it _still_ isnt auto-updating (most likely cuz our backend is down) then they should just manually trigger a one-off rebuild through the dashboard!